core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.ts

```diff
@@ -113,14 +114,18 @@ export class AudioBarComponent {
   }
 
   onNewLanguageSelected(): void {
-    this.audioTranslationLanguageService.setCurrentAudioLanguageCode(
-      this.selectedLanguage.value);
+    if (this.selectedLanguage.value !== null) {
```

Comment:
Should we do something when it is `null`?

Reply:
Done